### PR TITLE
magento/devdocs#: GraphQL functional testing. Add info about fixtures

### PR DESCRIPTION
**Pull request by atwixfirster**
## This PR is a:

- [X] Content update

## Summary

When this pull request is merged, it will update "GraphQL functional testing" page and add information about test fixtures. 

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dshevtsov** [comment]
@okolesnyk do we support new MTF tests for GraphQL?

File: guides/v2.3/graphql/functional-testing.md [hunk 1]
```diff
@@ -134,9 +144,26 @@ To use this fixture in a test, add it to the test's annotation in the following
...
+    /**
+     * @magentoApiDataFixture Magento/Checkout/_files/quote_with_simple_product_saved.php
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     */
+    public function testSetNewBillingAddressByRegisteredCustomer()
+    {
+      // Test body
+    }
+```
+
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: > It looks like we have two examples of invoking a single fixture. This example should show both invocations.

@dobooth , I hope I understood you correctly and I want clarify that in the first example `Magento/Customer/_files/customer.php` invokes in the `testCreateEmptyCart()` test and the second one in the another test - `testSetNewBillingAddressByRegisteredCustomer()`. Mentioned tests are not fake and they are exist in the Magento repository.

Thank you!

File: guides/v2.3/graphql/functional-testing.md [hunk 2]
```diff
@@ -85,10 +84,21 @@ The `\Magento\GraphQl\TestModule\GraphQlQueryTest.php` test case uses two test m
...
 * `TestModuleGraphQlQueryExtension` - This module extends `TestModuleGraphQlQuery`, adding the `integer_list` extension attribute. It's located at `<installdir>/dev/tests/api-functional/_files/TestModuleGraphQlQueryExtension`.
 
+## Creating fixtures
+
+Fixtures, which are part of the testing framework, prepare preconditions in the system for further testing. For example, when you test the ability to add a product to the shopping cart, the precondition is that a product must be available for testing. 
+
+A fixture consists of two files:
 
-## Using fixtures
+- The fixture file, which defines the test
```
[the remark this answers is not in the document]
Comment (keharper) [reply guides/v2.3/graphql/functional-testing.md]: Both bullets are sentence fragments, so no.

File: guides/v2.3/graphql/functional-testing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/graphql/functional-testing.md]: @atwixfirster Sorry I missed that functional code is the annotation, not the function itself. This is indeed correct as you wrote it!

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks, @atwixfirster.

**Pull request merged**